### ISSUE TITLE
Aerial seems to break basic navigation and folding?

### ISSUE THREAD

**Issue by windowsrefund** [AstroNvim/AstroNvim#762]
Please excuse me if this should be filed as a bug but I'm just getting a sense of what Aerial is and am unsure of what my expectations should be. That said, I'm really surprised to see basic vim navigation mappings like { and } not working in the classic sense. Normally, these keys are used to jump up and down by paragraph. With the stevearc/aerial.nvim plugin installed, I see they are bound to AerialNext and AerialPrev.  These functions just seem to jump to the next symbol. Honestly, it's really odd behavior.

The plugin also appears to interfere with folding in ways I can't really describe either.

Basically, I've chosen to disable it for now but I'd appreciate hearing what others experience.

**Comment by danilobjr** [comment]
I've just disabled Aerial as well. TBH I'll leave this way because I don't really use it. Now I can use `{` and `}` again. What a relief! 😌

**Comment by mehalter** [comment]
Yeah, I'm not a big fan of these bindings. I am planning on changing them to something less conflicting in `v2.0.0` release of AstroNvim. In the mean time if you want the symbol sidebar behavior or want to change these bindings manually, you can overwrite them in your `user/init.lua`. Here is an example `user/init.lua` that simply removes them:

```lua
return {
  plugins = {
    aerial = {
      on_attach = function() end,
    },
  },
}
```
